For AMD BSPs we solely rely on the pulseaudio systemd service
to start a server. So we disable PA autospawning by default.

diff --git a/src/pulse/client.conf.in2 b/src/pulse/client.conf.in
index 17753b0..6896640 100644
--- a/src/pulse/client.conf.in2
+++ b/src/pulse/client.conf.in
@@ -22,7 +22,7 @@
 ; default-server =
 ; default-dbus-server =
 
-; autospawn = yes
+autospawn = no
 ; allow-autospawn-for-root = no
 ; daemon-binary = @PA_BINARY@
 ; extra-arguments = --log-target=syslog
 
